cli/ceph/balancer.py

```diff
@@ -16,3 +16,83 @@ def status(self):
         if isinstance(out, tuple):
             return out[0].strip()
         return out
+
+    def set_state(self, state):
+        """
+        Sets balancer state to ON/OFF
+        Args:
+            state (str): state of balancer on/off
+        """
+        cmd = f"{self.base_cmd} {state}"
+        out = self.execute(sudo=True, check_ec=False, long_running=False, cmd=cmd)
+        if isinstance(out, tuple):
+            return out[0].strip()
+        return out
+
+    def mode(self, mode):
+        """
+        Sets the balancer mode
+        Args:
+            mode (str): mode to be set (crush-compat / upmap)
+        """
+        cmd = f"{self.base_cmd} mode {mode}"
+        out = self.execute(sudo=True, check_ec=False, long_running=False, cmd=cmd)
```

Comment:
Can we remove "check_ec=False, long_running=False" since by defult it is false and get parmeter from kw

https://github.com/red-hat-storage/cephci/blob/master/cli/__init__.py#L5

Example:https://github.com/red-hat-storage/cephci/blob/master/cli/ceph/mgr.py#L27